app/controllers/case_logs_controller.rb

```diff
@@ -127,15 +127,15 @@ def filtered_case_logs
       filters.each do |category, values|
         next if values.reject(&:empty?).blank?
 
-        query = query.public_send("filter_by_#{category}", values)
+        query = query.public_send("filter_by_#{category}", values, current_user)
```

Comment:
Can we make `current_user` the `value` here so we don't need to change the method signature?

Reply:
Uhm, would that mean putting an if statement like this or smth:
```
if category == 'user'
    values == values.include?(''all') ? "all" : current_user
end 
```

Looks wordy unless I'm misunderstanding something, besides default values in scopes sort of combat any massive issues with having 2 arguments in the signature 🤷‍♀️

Reply:
I was thinking you could do it based on what's in the view. Not super important though if it's fiddly.